internal/statemachine/image_definition.go

```diff
@@ -217,3 +220,42 @@ func newMissingURLError(context *gojsonschema.JsonContext, value interface{}, de
 type MissingURLError struct {
 	gojsonschema.ResultErrorFields
 }
+
+// generatePocketList returns a slice of strings that need to be added to
+// /etc/apt/sources.list in the chroot based on the value of "pocket"
+// in the rootfs section of the image definition
+func (ImageDef ImageDefinition) generatePocketList() []string {
+	pocketMap := map[string][]string{
+		"release": []string{},
+		"updates": []string{
```

Comment:
Could we reorder this for clarity, having release -> security -> updates -> proposed? It's a minor thing, but gives a better vibe. Also, this is this one time where in C I'd just use the evil fall-through capabilities of the `switch` statement :D (I'm evil, I know).

Reply:
Go actually has fallthrough as well! I thought this made more sense as a map however.

Reply:
Re-ordered!